README.md

```diff
@@ -38,14 +38,14 @@ git submodule update --init --recursive --depth 1
 
 To build the site in a container, run the following to build the container image and run it:
 
-```
+```bash
 make container-image
 make container-serve
 ```
 
 If you see errors, it probably means that the hugo container did not have enough computing resources available. To solve it, increase the amount of allowed CPU and memory usage for Docker on your machine ([MacOSX](https://docs.docker.com/docker-for-mac/#resources) and [Windows](https://docs.docker.com/docker-for-windows/#resources)).
 
-Open up your browser to http://localhost:1313 to view the website. As you make changes to the source files, Hugo updates the website and forces a browser refresh.
+Open up your browser to <http://localhost:1313> to view the website. As you make changes to the source files, Hugo updates the website and forces a browser refresh.
```

Comment:
Why adding the `<` and `>` characters? It will confuse the markdown parser which may believe it is a HTML tag!

Reply:
https://www.markdownguide.org/basic-syntax#urls-and-email-addresses

This is the syntax of markdown，For naked URL

Reply:
Okay ... since these README files are only for github view. I'm fine with this change.